tools/Google.Cloud.Tools.ReleaseManager/GitHelpers.cs

```diff
@@ -0,0 +1,49 @@
+﻿using LibGit2Sharp;
```

Comment:
nit: Missing license? Not sure if these files have it.

Reply:
Doh, absolutely. Fixing now, thanks.